lib/radiator_web/graphql/admin/resolvers/editor.ex

```diff
@@ -302,6 +307,10 @@ defmodule RadiatorWeb.GraphQL.Admin.Resolvers.Editor do
     {:ok, Media.NetworkImage.url({network.image, network})}
   end
 
+  def get_image_url(%User{person: person}, _, _) do
+    {:ok, Media.PersonAvatar.url({person.avatar, person})}
```

Comment:
I did add an `image_url` function to the ecto schema and bounced through there always to keep the layering more layered.